rust/cloud-storage/document_storage_service/src/api/documents/quick_create_task/quick_create_task_handler.rs

```diff
@@ -0,0 +1,250 @@
+use axum::{Extension, extract::State, http::StatusCode, response::IntoResponse};
```

Comment:
this is a ton of code duplication from POST /documents and will require us to remember to update this and the original create document endpoint should there be changes.

this also increases the likely-hood of creating an invalid document states as they are separated so we may mess something up unintentionally